src/Tools/CommercialData/CommercialData.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './CommercialData.scss';
+import { connect } from 'react-redux';
+import { getAppropriateAuthToken } from '../../App';
+
+const CommercialData = ({ authToken }) => {
```

Comment:
I would suggest renaming the 3 files that you added, since we already have the "old" `CommercialData.jsx` in `CommercialDataPanel`  and I would like to avoid some confusion. I would suggest something like `RapidResponseDesk.jsx` so it would also be more in the line of the official name of the feature (please also change the `id` of the tab and add new constant in `TABS.RAPID_RESPONSE_DESK`)